src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session_FilterModel.cs

```diff
@@ -221,12 +221,20 @@ internal partial class Session
 
             private Boolean IsAfterDot(Model model, ITextSnapshot textSnapshot, Dictionary<TextSpan, string> textSpanToText)
             {
-                var span = model.OriginalList.Span;
+                var originalSpan = model.OriginalList.Span;
 
                 // Move the span back one character if possible.
-                span = TextSpan.FromBounds(Math.Max(0, span.Start - 1), span.End);
+                var span = Span.FromBounds(Math.Max(0, originalSpan.Start - 1), originalSpan.End);
 
-                var text = model.GetCurrentTextInSnapshot(span, textSnapshot, textSpanToText);
+                // Because we are adjusting the span, it's not safe to call
+                // model.GetCurrentTextInSnapshot. GetCurrentTextInSnapshot starts by
+                // mapping the span into the ViewBuffer. In debugger intellisense, if the
```

Comment:
The helper we used to call mapped the span passed in onto the view buffer. If the position was at the start of the debugger intellisense view, moving the position back one produced a span that could not be mapped onto the view buffer. This appears in debugger intellisense because that's how we've constructed the projection.

Since this helper is just trying to look at the buffer contents, it was probably never right for it to be mapping spans onto the view.